docs/chat-configuration/ui-customization/incoming-message/readmore-article.md

```diff
@@ -36,6 +36,21 @@ val customProvider = ChatUIProvider(context).apply {
         articleUIConfig.verticalMargin = x to y // Pair of top, bottom margin values, in pixels
 
         channelsUIProvider = ... // A QuickOptionUIProvider object
+        
+        // Configure the main background of the Article
+        background = ....
+
+         // Configure the article title
+        title.apply {
+            font = ... // A StyleConfig object
+            background = ... // A color int
+        }
+
+        // Configure the article body
+        body.apply {
+            setStyle( .. ) // Set the font using its font family (capable with css)
```

Comment:
the comment should be changed
"Sets the body font style by defining: size, color, fontFamily and fontStyle"